### PR TITLE
CA-320182: Error in adding vGPU in New VM wizard.

### PR DESCRIPTION
**Pull request by michael2012z**
Signed-off-by: Michael Z <michael.zhao@citrix.com>

### CODE REVIEW

File: XenModel/XenAPI/FriendlyErrorNames.resx [hunk 1]
```diff
@@ -1959,7 +1959,7 @@ Authorized Roles: {1}</value>
     <value>The VGPU is not compatible with any PGPU in the destination.</value>
   </data>
   <data name="VGPU_TYPE_NOT_COMPATIBLE" xml:space="preserve">
-    <value>You try to create a VGPU that is not compatible with existing types on the VM.</value>
+    <value>You tried to create a VGPU that is not compatible with existing types on the VM.</value>
```
Comment (MihaelaStoica) [inline XenModel/XenAPI/FriendlyErrorNames.resx]: We should avoid using "VGPU" in the user-facing messages, so here we should say "virtual GPU" .
My suggestion for this message was (note "virtual GPU" and "the existing types"):
"You tried to create a virtual GPU that is not compatible with the existing types on the VM."

File: XenAdmin/Dialogs/AddVGPUDialog.cs [hunk 2]
```diff
@@ -91,7 +91,7 @@ private void PopulateComboBox()
                         comboBoxTypes.Items.Add(new GpuTuple(gpu_group, vgpuType, disabledTypes.ToArray())); // GPU_type item
 
                     if (commonTypes.Count == 1 && !disabledTypes.Contains(commonTypes[0]))
-                        comboBoxTypes.SelectedItem = comboBoxTypes.Items[1]; // Items[0] is group
+                        comboBoxTypes.SelectedItem = comboBoxTypes.Items[0];
```
Comment (MihaelaStoica) [inline XenAdmin/Dialogs/AddVGPUDialog.cs]: So what gets selected here if Items[0] is a group?

Comment (michael2012z) [reply XenAdmin/Dialogs/AddVGPUDialog.cs]: If host driver was not installed, Items[0] is a device (no group).
If host driver was installed, Items[0] is the group, but the combobox handles this, focus slip to the next enabled item.

File: XenModel/XenAPI/FriendlyErrorNames.resx (hunk 1 above)
Comment (michael2012z) [reply XenModel/XenAPI/FriendlyErrorNames.resx]: I've submit a PR in XAPI to update the text.

**Comment by michael2012z** [comment]
@MihaelaStoica Any more comments for this PR? We can update Xapi error text with another PR when it's ready in XAPI build.

**Pull request merged**